OpenEdXMobile/src/test/java/org/edx/mobile/profiles/UserProfileFragmentTest.java

```diff
@@ -16,8 +16,10 @@
 import java.util.Collections;
 import java.util.List;
 
-import static org.assertj.core.api.Java6Assertions.assertThat;
 import static org.assertj.android.api.Assertions.assertThat;
+import static org.assertj.core.api.Java6Assertions.assertThat;
```

Comment:
Do we still need `Java6Assertions`?

Reply:
@miankhalid Yes, we are using `Java6Assertions.assertThat` to check the `FragmentUserProfileBinding` should not null

Reply:
Can't we replace those usages with the new `Assertions.assertThat`?

Reply:
@miankhalid we can use Assert.assertNotNull.